sunpy/coordinates/transformations.py

```diff
@@ -66,6 +68,69 @@
            'hme_to_gei', 'gei_to_hme', 'gei_to_gei']
 
 
+# Boolean flag for whether to ignore the motion of the center of the Sun in inertial space
+_ignore_sun_motion = False
+
+
+@contextmanager
+def transform_with_sun_center():
+    """
+    Context manager for coordinate transformations to ignore the motion of the center of the Sun.
+
+    Normally, coordinates refer to a point in inertial space (relative to the barycenter of the
+    solar system).  Transforming to a different observation time does not move the point at all,
+    but rather only updates the coordinate representation as needed for the origin and axis
+    orientations at the new observation time.  However, the center of the Sun moves over time.
+    Thus, a coordinate that points to a solar feature (e.g., the center of the Sun) will not point
+    to the same solar feature at other observation times.
```

Comment:
I think this needs tightening up a bit, because even with coordinates of a solar feature (e.g. active region) will not point to the solar feature at a later time due to solar rotation. Maybe

"Thus, a coordinate on the surface of the Sun will not remain on the surface of the Sun at other observation times"

Reply:
Edited